events/models.py

```diff
@@ -422,12 +423,14 @@ def __finish_with_outcome(self, outcome):
                     quantity=bet.has,
                     price=self.PRIZE_FOR_WINNING
                 )
-            # TODO: tutaj wallet change
-            #  bet.user.portfolio_value -= bet.has
             bet.user.save()
             # This cause display event in "latest outcome"
             bet.is_new_resolved = True
             bet.save()
+        # always -- portfolio_value
+        for transaction in Transaction.objects.filter(event=self):
```

Comment:
Should be for user in UsersPortfolio loop, it will be more efficient.  And I'm not sure is this correct calculation?